src/Presenters/ApiPresenter.php

```diff
@@ -71,30 +73,28 @@ public function run(Request $request): IResponse
 
         $api = $this->getApi($request);
         $handler = $api->getHandler();
+
         $authorization = $api->getAuthorization();
         $rateLimit = $api->getRateLimit();
 
-        $authResponse = $this->checkAuth($authorization);
```

Comment:
preco sme presunuli autorizaciu az za parametre a rate limit?

Reply:
Lebo treba spracovat parametre aby som mal dostupny Context do erroru. Nepacilo sa mi to ale lepsie som nevymyslel.

Reply:
Hm už si spomínam. Len teraz úplne zmeníme výstupy - ak niečo predtým padlo už na autorizácii, alebo rate limite parametre sa vôbec neriešili. Teraz to bude naopak a neviem povedať ci korektne.